src/components/FilterBar/FilterDisplay/Inputs/DateInputRelative.react.js

```diff
@@ -0,0 +1,139 @@
+export class DateInputRelative extends React.Component {
```

Comment:
Think `RelativeDataInput` would be more consistent with existing component naming.